src/System.Private.ServiceModel/System.Private.ServiceModel.sln

```diff
@@ -120,10 +120,6 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Unit", "Unit", "{19B3A13A-9
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "UnitTests.Common", "tests\Common\Unit\UnitTests.Common.csproj", "{E896294A-AB4A-4AF5-A01C-A19E3972EFF9}"
 EndProject
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "NetHttp", "NetHttp", "{1A73BF1C-F8DB-47EC-8198-2A6DFFC93333}"
```

Comment:
I see the removed projects.  Should there also be some added projects for them too?